xbmc/Util.cpp

```diff
@@ -929,8 +926,7 @@ bool CUtil::CreateDirectoryEx(const std::string& strPath)
   }
 
   // was the final destination directory successfully created ?
-  if (!CDirectory::Exists(strPath)) return false;
-  return true;
+  return CDirectory::Exists(strPath);
```

Comment:
These are here for debugging reasons, e.g. you put the breakpoint on either of those.

Reply:
If I need to debug stuff like this I quickly any temporarily change the code so that I can set a break point. No need to have this "debug helper" in production code, imo. OTOH, no hard feelings. Just saying.